Makefile

```diff
@@ -0,0 +1,22 @@
+
+
+# Command for the initial run of the bot, scrapes Documentation and then starts.
+firstrun:
+	echo "Scraping documentation data..."
+	$(MAKE) scrape
+	echo "Installing requirements..."
+	python3 -m pip install -r requirements.txt
+	echo "Starting Bot..."
+	python3 src/bot/run.py
+
+# Simple shortcut for starting the bot.
+run: 
+	python3 src/bot/run.py
+
+# Run all scraping spiders and put their results into the data/ directory
+scrape:
+	mkdir -p data/
+	rm -rf data/*
+	cd src/ && \
+	scrapy crawl cpp-symbols --loglevel INFO -o ../data/cpp_symbols.json && \
+	scrapy crawl cpp-stubs   --loglevel INFO -o ../data/cpp_stubs.json
```

Comment:
Well yeah, this isn't a make problem at all. This is just a series of steps that need to be executed before each invocation.

In fact, you could probably integrate this at some level into the package itself. (Maybe via command line flags, or defaults or something).